### PR TITLE
Add a Logo component

### PR DESCRIPTION
**Pull request by colophonemes**

### CODE REVIEW

**Comment by jpaddison3** [comment]
What logo url are you using?

**Comment by colophonemes** [comment]
Sorry:
```
https://images.ctfassets.net/ohf186sfn6di/7J4cBC9SXCWMoqqCIqI0GI/affe205261bb8cff47501a0ada0f2268/ea-logo-square-1200x1200__1_.png?h=100
```

File: packages/lesswrong/components/common/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { registerComponent, getSetting } from 'meteor/vulcan:core';
+import { withStyles, withTheme } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    height: 48
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/Logo.jsx]: Can JP ask some dumb questions? What are the units for height? What does root mean in this context?

File: packages/lesswrong/components/common/Header.jsx [hunk 2]
```diff
@@ -67,7 +69,8 @@ const styles = theme => ({
   rightHeaderItems: {
     marginRight: -theme.spacing.unit,
     display: "flex",
-  }
+  },
+
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/Header.jsx]: nitpick, no extra line

File: packages/lesswrong/components/common/Header.jsx [hunk 3]
```diff
@@ -48,6 +48,8 @@ const styles = theme => ({
     verticalAlign: 'middle',
     fontSize: 19,
     position: "relative",
+    display: 'flex',
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/Header.jsx]: Another dumb question / I guess I just want to completely grok all the css we put in: What does this do?

File: packages/lesswrong/components/common/Logo.jsx (hunk 1 above)
Comment (colophonemes) [reply packages/lesswrong/components/common/Logo.jsx]: Material UI's default unit is pixels, so `Number`s are cast to `Npx`

Root is just the classes I'm applying to the root component. As far as I can tell, it's an arbitrary name, but following a convention from other components

**Pull request merged**